flink-metrics/flink-metrics-dropwizard/src/test/java/org/apache/flink/dropwizard/ScheduledDropwizardReporterTest.java

```diff
@@ -124,6 +129,89 @@ public void testAddingMetrics() throws NoSuchFieldException, IllegalAccessExcept
 		metricRegistry.shutdown();
 	}
 
+	@Test
```

Comment:
Can you add a high level description about what this test does please? Otherwise this looks good.